### PR TITLE
Update language latin/es

### PR DESCRIPTION
**Pull request by RockLee-BC**

### CODE REVIEW

File: Themes/default/languages/LightPortal/.spanish_es.php [hunk 1]
```diff
@@ -54,25 +57,44 @@
...
 $txt['lp_extra'] = 'Páginas y bloques';
 $txt['lp_extra_info'] = 'Aquí puede encontrar configuraciones generales de páginas y bloques.';
 
 $txt['lp_show_tags_on_page'] = 'Mostrar palabras clave en la parte superior de la página';
 $txt['lp_show_comment_block'] = 'Mostrar bloque de comentarios';
-$txt['lp_show_comment_block_set'] = array('none' => 'Ninguno', 'default' => 'Integrado');
-$txt['lp_num_comments_per_page'] = 'Número de comentarios de los padres por página'; // Check meaning
-$txt['lp_page_editor_type_default'] = 'El tipo de editor de la página por defecto';
+$txt['lp_disabled_bbc_in_comments'] = 'BBC permitidos en los comentarios';
+$txt['lp_show_comment_block_set'] = array('ninguno' => 'Ninguno', 'por defecto' => 'Integrado');
```
Comment (dragomano) [inline Themes/default/languages/LightPortal/.spanish_es.php:67]: Please, do not translate array keys, it's a bad idea.

File: Themes/default/languages/LightPortal/.spanish_es.php [hunk 2]
```diff
@@ -175,13 +198,20 @@
...
 	'show_author_and_date' => 'Mostrar el autor y la fecha de creación',
 	'allow_comments' => 'Permitir comentarios'
 );
 
+// Tabs
+$txt['lp_tab_content'] = 'Contenido';
+$txt['lp_tab_seo'] = 'SEO';
+$txt['lp_tab_access_placement'] = 'El acceso y la colocación';
+$txt['lp_tab_appearance'] = 'Apariencia';
+$txt['lp_tab_tuning'] = 'Tuning'; // I don't understand the context
```
Comment (dragomano) [inline Themes/default/languages/LightPortal/.spanish_es.php:213]: The tuning tab contains specific settings of the current block.

File: Themes/default/languages/LightPortal/.spanish_es.php (hunk 1 above)
Comment (RockLee-BC) [reply Themes/default/languages/LightPortal/.spanish_es.php:67]: You are right I did not realize, I modify it!

File: Themes/default/languages/LightPortal/.spanish_es.php (hunk 2 above)
Comment (RockLee-BC) [reply Themes/default/languages/LightPortal/.spanish_es.php]: Great gives me an idea how to put it, thanks!

**Pull request merged**